client/src/main/java/io/pravega/client/stream/ReaderGroupConfig.java

```diff
@@ -177,6 +213,20 @@ public ReaderGroupConfigBuilder startFromCheckpoint(final Checkpoint checkpoint)
            return this;
        }
 
+       /**
+        * Set the retention config for the {@link ReaderGroup}.
+        * For Consumption based retention of data in the Stream, this field should be set to
+        * CONSUMPTION_BASED_USER_STREAMCUT or CONSUMPTION_BASED_AT_LAST_CHECKPOINT.
```

Comment:
```suggestion
```
Again it is valid for a stream to enable CBR and for one reader group to not take advantage of it.

Reply:
Yes it is completely valid. That would be a non-subscriber Reader Group.
See: https://github.com/pravega/pravega/wiki/PDP-47:-Pravega-Streams:-Consumption-Based-Retention

Reply:
Ok. Then please remove these two lines.

Reply:
I have changed this based on recent discussions. I think this explanation is necessary to give the user a correct understanding of when RG reads will impact truncation and when not.